### DISCUSSION TITLE
Norns on a Raspberry Pi without norns shield

### DISCUSSION THREAD

**Discussion by bcostaaa01** [monome/norns#1661] [Q&A] [answered]
Hey norns dev team 👋

I have been planning my adventure with a Raspberry Pi and norns, but don't have the norns shield... 

Am I still able to use norns with the Raspberry Pi and build a custom hardware for it, or do I need to get something like the norns shield PCB: https://pushermanproductions.com/product/norns-shield-v210330-smd-pre-soldered-pcb/

I have been thinking about this, but have not found anything as specific as the information I need to assess whether I am able to make a custom hardware for a Raspberry Pi or if I really need the norns shield PCB...🤔

If I was to build a custom hardware without shield, what hardware components would I need and how would I set them all up to work perfectly with norns?

I look forward to hearing from you, as well as from anyone in the community who is trying to achieve the same result!

**Comment by tehn** [comment, marked as answer]
Please see here: https://github.com/monome/norns-shield
